dlls/geoip/geoip_natives.cpp

```diff
@@ -117,6 +117,7 @@ static cell AMX_NATIVE_CALL amx_geoip_country(AMX *amx, cell *params)
 
 	if (!country)
 	{
+		MF_SetAmxString(amx, params[2], "error", params[3]); // to keep compatibility with previous version
 		return 0;
```

Comment:
Incorrect return value for bcompat, the old version actually uses the `MF_SetAmxString` return value (and thus `strlen("error")`). I wouldn't consider this sensible behavior so maybe an `_ex` variant with a boolean return value is in order? 

Also it looks like multiple function docs need to be fixed as many say that `0` will be returned on unsuccessful lookup which is incorrect.